readme.md

```diff
@@ -711,6 +711,51 @@ Config Based Styles Can be Applied to Components.
     ]
 }
 
+### Query Param Support With React Crux
+We can give queryParams in config. It will append queryparams with every fetch call.
+
```

Comment:
please mention whether additional models will also use these params or is it only applicable for the main model

Reply:
If needed we can also use for it.
It can be used for all the models. It will just add queryParams for all the request url